k8s/stubbornjava.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: stubbornjava-deployment
+  labels:
+    app: sj-web
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sj-web
+  template:
+    metadata:
+      labels:
+        app: sj-web
+    spec:
+      containers:
+        - name: sj-web
+          image: containers.pkg.github.com/stubbornjava/stubbornjava-webapp:prod
```

Comment:
At some point you'll probably want to extract these manifests into a helm chart and provide things like `image` with it's tag as a value that's passed in.